tests/models/test_vllm.py

```diff
@@ -17,15 +20,15 @@ class TEST_VLLM:
     except ModuleNotFoundError:
         pass
     torch.use_deterministic_algorithms(True)
-    tasks.initialize_tasks()
-    multiple_choice_task = tasks.TASK_REGISTRY.get("arc_easy")()  # type: ignore
+    task_list = task_manager.load_task_or_group(["arc_easy", "gsm8k", "wikitext"])
```

Comment:
what's the difference between `load_task_or_group` versus `get_task_dict` ? 


do we have a list of the ways a user (should, or can) interact with the `TaskManager` ?

Reply:
Both are used, `get_task_dict` calls `load_task_or_group`. I've adjusted so that if the user uses `simple_evaluate` they won't need to interface with `TaskManager`. It's mostly used in `__main__.py` to help list out valid stock tasks or include tasks outside of stock tasks.